third_party/ameba/mbedtls.cmake

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.6)
 
 project(mbedtls)
 
-set(dir "${sdk_root}/component/common/network/ssl/mbedtls-matter")
+set(dir "${sdk_root}/component/common/application/matter/mbedtls/")
```

Comment:
Is this ok? Since docker image update is async from the main build.
I believe this may need moving to a PR where 0.6.28 starts to be used (after it is fully built)

Reply:
Agree with @andy31415 the path is [still valid](https://github.com/pankore/ambd_sdk_with_chip_non_NDA/tree/ameba_update_2022_12_21/component/common/network/ssl/mbedtls-2.4.0) in latest tag

Reply:
I asked because the Ameba build is failing in this PR, so I assumed that is the reason. Docker images should not fail compilation PRs. I believe this change should be done in a separate update.